github/github.go

```diff
@@ -888,9 +888,9 @@ func formatRateReset(d time.Duration) string {
 	if isNegative {
 		d *= -1
 	}
-	secondsTotal := int(d.Seconds())
-	minutes := int((secondsTotal) / 60)
-	seconds := int(secondsTotal - minutes*60)
+	secondsTotal := int(0.5 + d.Seconds())
+	minutes := (secondsTotal) / 60
```

Comment:
Already. You did the review much faster than I typed `git commit --amend` in terminal.